### ISSUE TITLE
Remove 'path' from all wallet function arguments

### ISSUE THREAD

**Issue by cygnet3** [cygnet3/sp-client#6]
This is a general comment for every function that has `path` and `label` as arguments: I think adding `label` as an argument makes sense, this would allow us to have multiple wallets in memory later on. But I think it's better if we let the user decide the `path` only once, for example when creating the `sp_client` object

_Originally posted by @cygnet3 in https://github.com/cygnet3/sp-backend/pull/5#discussion_r1432430565_

**Comment by Sosthene00** [comment]
I think I know how to do it, we have the dir in the `root` member of Nakamoto config, so let's read it.

**Comment by cygnet3** [comment]
No longer relevant
